sell_only_by_packaging/models/product_template.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    sell_only_by_packaging = fields.Boolean(
+        string="Only sell by packaging",
+        company_dependent=True,
+        default=False,
+        help="Restrict the usage of this product on sale order lines without "
+        "packaging defined",
+    )
+
+    min_sellable_qty = fields.Float(
+        compute="_compute_template_min_sellable_qty",
+        help=(
+            "Minimum sellable quantity, according to the available packagings, "
+            "if Only Sell by Packaging is set."
+        ),
+    )
+
+    @api.depends(
+        "sell_only_by_packaging",
+        "uom_id.factor",
+        "product_variant_ids.min_sellable_qty",
+    )
+    def _compute_template_min_sellable_qty(self):
+        for record in self:
+            record.min_sellable_qty = 0.0
+            if len(record.product_variant_ids) == 1:
```

Comment:
```suggestion
            if record.product_variant_count == 1:
```

Reply:
What would we gain from triggering a compute method?